x-pack/plugins/security_solution/public/common/containers/events/last_event_time/index.ts

```diff
@@ -71,7 +72,7 @@ export const useTimelineLastEventTime = ({
 
   const timelineLastEventTimeSearch = useCallback(
     (request: TimelineEventsLastEventTimeRequestOptions) => {
-      let didCancel = false;
+      didCancel.current = false;
       const asyncSearch = async () => {
         abortCtrl.current = new AbortController();
```

Comment:
Do you think we should `abortCtrl.current.abort();` also before initiating a new request?